lib/tasks/static_pages.rake

```diff
@@ -20,7 +20,7 @@ namespace :static_pages do
       outpath = File.join([Rails.root, 'public', output])
       resp = app.get(route)
       if resp == 200
-        File.delete(outpath) unless not File.exist?(outpath)
+        File.delete outpath if File.exist?(outpath)
```

Comment:
All I can not succeed at deducting from this conversation is "Buffalo buffalo Buffalo buffalo buffalo buffalo Buffalo buffalo"